README.md

```diff
@@ -44,6 +44,15 @@ Pipe the output of `terraform plan` into `landscape` to reformat the output.
 terraform plan ... | landscape
 ```
 
+## Docker
+
+Build the docker image using provided Dockerfile and use it directly:
+
+```bash
+docker build . -t landscape
+terraform plan ... | docker run landscape
```

Comment:
Minor nit: this will leave containers on your filesystem after the process exits. You want the `--rm` flag to follow best practices:

```
terraform plan ... | docker run --rm landscape
```